app/assets/stylesheets/styles/ui/inputs/inputs.scss

```diff
@@ -15,3 +15,7 @@ input[type="color"],
 textarea {
   border-radius: $global-radius;
 }
+
+input.ng-invalid {
+  border: 1px solid red;
```

Comment:
I think (from memory) there's an $of-red variable (check _variables.scss). There might even be an $of-alert - or something similar - color.